tests/nav-spine_in-spine-no-list-style/OPS/package.opf

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Navigation Documents</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:creator>Dave Cramer</dc:creator>
```

Comment:
Why is Dave in this one? I suspect you branched from one of his tests, but in that case he's an author of every test :)

Reply:
ok